libraries/Microsoft.Bot.Builder.AI.QnA/QnAMakerEndpoint.cs

```diff
@@ -40,6 +40,15 @@ public QnAMakerEndpoint(QnAMakerService service)
         [JsonProperty("knowledgeBaseId")]
         public string KnowledgeBaseId { get; set; }
 
+        /// <summary>
+        /// Gets or sets the QnAServiceType.
+        /// </summary>
+        /// <value>
+        /// QnAServiceType V2 or language.
+        /// </value>
+        [JsonProperty("qnAServiceType")]
+        public string QnAServiceType { get; set; }
```

Comment:
Consider using a constant to define the valid types

Reply:
enum?